### PR TITLE
[TRUS-4267] - govuk migration

### PR DESCRIPTION
**Pull request by jassalrichy**
Move maintain-protectors-frontend to Play-front-end govuk and Play-frontend-HMRC

### CODE REVIEW

File: app/views/ViewUtils.scala [hunk 1]
```diff
@@ -26,6 +28,39 @@ object ViewUtils {
...
   def breadcrumbTitle(title: String)(implicit messages: Messages): String = {
-    s"$title - ${messages("site.service_section")} - ${messages("site.service_name")} - GOV.UK"
+    s"$title - ${messages("site.service_section")} - ${messages("service.name")} - GOV.UK"
   }
+
+  def errorHref(error: FormError, radioOptions: Seq[RadioOption] = Nil): String = {
+    error.args match {
+      case x if x.contains("day") || x.contains("month") || x.contains("year") =>
+        s"${error.key}.${error.args.head}"
+      case _ if error.message.toLowerCase.contains("yesno") =>
```
Comment (adamconder) [inline app/views/ViewUtils.scala:38]: This has just been changed by @carrielies for maintain-a-protector-frontend

File: app/views/components/SubHeading.scala.html [hunk 2]
```diff
@@ -14,11 +14,13 @@
...
+@this()
 
-<div class="section">
-    <input type="@inputType" class="button" value="@messages(messageKey)" />
-</div>
+@(key: String, param: String, hiddenText: Boolean = true)(implicit messages: Messages)
+
+<span class="govuk-caption-l">
+ @if(hiddenText){
+  <span class="govuk-visually-hidden">@messages(s"$key.hidden")</span>
```
Comment (adamconder) [inline app/views/components/SubHeading.scala.html:23]: The message key for this is not defined in messages.en or .cy

File: app/views/ViewUtils.scala (hunk 1 above)
Comment (jassalrichy) [reply app/views/ViewUtils.scala:38]: Updated

File: app/views/components/SubHeading.scala.html (hunk 2 above)
Comment (jassalrichy) [reply app/views/components/SubHeading.scala.html]: Component not used. Updated to match SubHeading component within register-trust-other-individual-frontend:
https://github.com/hmrc/register-trust-other-individual-frontend/blob/master/app/views/components/SubHeading.scala.html

**Pull request merged**